src/plugins/validation/2and3/semantic-validators/operation-ids.js

```diff
@@ -64,11 +64,13 @@ module.exports.validate = function({ resolvedSpec }) {
     // - paths with path param has a GET, a DELETE, and a POST or PUT or PATCH.
```

Comment:
The operationdId convention implemented here is not correct.  The rule should be:
- If the path does not _end_ in a path parameter:
  - `POST` should be "create" or "add"
  - "GET" should be "list"
- If the path _ends_ in a path parameter:
  - if the path has a `PATCH` operation
     - `PATCH` should be "update"
  - else if the the path has a "POST" operation
      - `POST` should be "update"
  - `PUT` should be "replace"
  - `DELETE` should be `delete`

So two important things to note:
1. You are only concerned with a path parameter _at the end_ of the path
1. You can't check operations in isolation -- you have to check all operations on the path together.